### PR TITLE
martinb3/docker fixes

### PR DESCRIPTION
**Pull request by martinb3**
Per #27, clean up the Docker setup here, and make sure it works without a lot of special arguments/settings.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
+RUN pip3 install --user -r test-requirements.txt && pip3 install --user -r requirements.txt && pip3 install --user -e .
 
-RUN mkdir -p ${APP_HOME}
-ADD . ${APP_HOME}
-
-RUN cd ${APP_HOME} && pip3 install --user -r test-requirements.txt -e .
-
-ENTRYPOINT [ "python3", "tuvok/tuvok/cli.py" ]
+# just try to run tuvok, so we can be sure it is installed correctly
+RUN tuvok --version
```
Comment (jpbochi) [inline Dockerfile]: do we really need this line, since we have tests and all?

Comment (martinb3) [reply Dockerfile]: No, but I wanted it to fail faster than waiting for CircleCI to spin up a new container for tests.

**Comment by michaelmoussa** [comment]
@martinb3 ran a new build successfully but no tests run when I execute:

```bash
$ docker run tuvok -c "cd /tuvok && pytest -v"
============================= test session starts ==============================
platform linux -- Python 3.7.1, pytest-3.10.1, py-1.7.0, pluggy-0.8.0 -- /usr/local/bin/python
cachedir: .pytest_cache
rootdir: /, inifile: tox.ini
collecting ... collected 0 items

========================= no tests ran in 0.01 seconds =========================
```

**Comment by martinb3** [comment]
> One other ask for the docs would be a callout that jq and json2hcl are not actually installed when you run setup.py and will have to installed separately if you do not use the docker file.

I also did some work recently to expose a missing jq/json2hcl command directly in the error messages when checks fail. I'll throw up a new issue for this. 👍

**Pull request merged**